app/index.ts

```diff
@@ -17,33 +17,31 @@ const SETTINGS_FILE = "settings.json";
 let settings = loadSettings();
 applyTheme(settings.foreground);
 
-function updateClock(evt) {
+clock.ontick = evt => {
```

Comment:
Because the `tsconfig.json` created by fitbit cli has some strict rules by default, you can't have _implicit `any`_. So just assign the function to the event handler and the type definition gives it the correct `TickEvent` type.